### ISSUE TITLE
Latest releases not published?

### ISSUE THREAD

**Issue by jacobras** [alorma/Compose-Settings#209]
It seems the latest available version if 1.0.3 (https://mvnrepository.com/artifact/com.github.alorma/compose-settings-ui-m3) while the Releases page shows several releases up to 1.0.9. I noticed those releases failed to publish, any plan to get them out? Would love to use this library in more projects now that it's multiplatform supported.

**Comment by alorma** [comment]
I didn't have time to fix the release process.

If anyone can help on fixing it. I would appreciate that.

**Comment by jacobras** [comment]
I have some experience publishing multiplatform libraries. For example, here's how I publish one of my libraries:
https://github.com/jacobras/ComposeActionMenu/blob/main/.github/workflows/publish.yml
https://github.com/jacobras/ComposeActionMenu/blob/main/compose-action-menu/build.gradle.kts

I use the `com.vanniktech.maven.publish` plugin, but of course the one you're using currently should also do the trick. The error on the last publish action run (https://github.com/alorma/Compose-Settings/actions/runs/8048761976/job/21985335767) confuses me, as `gradlew` is clearly available in the right place.

**Comment by jacobras** [comment]
@alorma Update: I took another look. The publish workflow fails because you're not checking out the code :) When you split up into different jobs, they each run in their own isolated area. Also, they run in parallel instead, unless you specifically link one with a `needs` dependency on the other.

I would suggest for the publishing workflow to have just one job. In it, just run checkout, JDK setup, publish command (which triggers the build anyway as a dependency).

Basically how I'm doing it in https://github.com/jacobras/ComposeActionMenu/blob/main/.github/workflows/publish.yml. Let me know if you need help! 🚀

**Comment by alorma** [comment]
Fixed!
